### ISSUE TITLE
ppc64el: build failure

### ISSUE THREAD

**Issue by fnordahl** [openstack-charmers/octavia-diskimage-retrofit#4]
We are currently cheating by building ``qemu`` with the ``--disable-blobs`` option and then including the relevant bits in stage packages.

Apparently there exist no stage package we can use for this blob, so we need to figure out how to retrieve and build the ``qemu`` blobs in the snap.

```
/build/octavia-diskimage-retrofit/stage/bin/qemu-system-ppc64 \
    -global virtio-blk-pci.scsi=off \
    -no-user-config \
    -enable-fips \
    -nodefaults \
    -display none \
    -machine pseries,accel=kvm:tcg \
    -m 1024 \
    -no-reboot \
    -rtc driftfix=slew \
    -kernel /var/tmp/.guestfs-0/appliance.d/kernel \
    -initrd /var/tmp/.guestfs-0/appliance.d/initrd \
    -object rng-random,filename=/dev/urandom,id=rng0 \
    -device virtio-rng-pci,rng=rng0 \
    -device virtio-scsi-pci,id=scsi \
    -drive file=/tmp/libguestfsNDwEYn/devnull1.img,cache=writeback,format=raw,id=hd0,if=none \
    -device scsi-hd,drive=hd0 \
    -drive file=/var/tmp/.guestfs-0/appliance.d/root,snapshot=on,id=appliance,cache=unsafe,if=none,format=raw \
    -device scsi-hd,drive=appliance \
    -device virtio-serial-pci \
    -serial stdio \
    -chardev socket,path=/tmp/libguestfsqVCVX3/guestfsd.sock,id=channel0 \
    -device virtserialport,chardev=channel0,name=org.libguestfs.channel.0 \
    -append "panic=1 console=hvc0 console=ttyS0 edd=off udevtimeout=6000 udev.event-timeout=6000 no_timer_check printk.time=1 cgroup_disable=memory usbcore.nousb cryptomgr.notests tsc=reliable 8250.nr_uarts=1 root=/dev/sdb selinux=0 guestfs_verbose=1 TERM=linux"
Could not access KVM kernel module: No such file or directory
qemu-system-ppc64: failed to initialize KVM: No such file or directory
qemu-system-ppc64: Back to tcg accelerator
qemu-system-ppc64: Could not find LPAR rtas 'spapr-rtas.bin'
```

**Comment by fnordahl** [comment]
Including the blobs in the snap leads to this failure:
```
qemu-system-ppc64: warning: Failed to set KVM's VSMT mode to 8 (errno -22)
qemu-system-ppc64: KVM implementation does not support Transactional Memory, try cap-htm=off
```

**Comment by fnordahl** [comment]
Fixed in commit c2f97ed489c93b63319b08293e85ba34aad3f221 and commit 8a61d6002ccd093a488c201763f0e426b185ad13
